include/cantera/thermo/PengRobinson.h

```diff
@@ -0,0 +1,359 @@
+//! @file PengRobinson.h
+
+// This file is part of Cantera. See License.txt in the top-level directory or
+// at https://cantera.org/license.txt for license and copyright information.
+
+#ifndef CT_PENGROBINSON_H
+#define CT_PENGROBINSON_H
+
+#include "MixtureFugacityTP.h"
+#include "cantera/base/Array.h"
+
+namespace Cantera
+{
+/**
+ * Implementation of a multi-species Peng-Robinson equation of state
+ *
+ * @ingroup thermoprops
+ */
+class PengRobinson : public MixtureFugacityTP
+{
+public:
+    //! @name Constructors and Duplicators
+    //! @{
+
+    //! Base constructor.
+    PengRobinson();
+
+    //! Construct and initialize a PengRobinson object directly from an
+    //! ASCII input file
+    /*!
+     * @param infile    Name of the input file containing the phase YAML data
+     *                  to set up the object
+     * @param id        ID of the phase in the input file. Defaults to the empty
+     *     string.
+     */
+    PengRobinson(const std::string& infile, const std::string& id="");
+
+    virtual std::string type() const {
+        return "PengRobinson";
+    }
+
+    //! @name Molar Thermodynamic properties
+    //! @{
+
+    virtual double cp_mole() const;
+    virtual double cv_mole() const;
+
+    //! @}
+    //! @name Mechanical Properties
+    //! @{
+
+    //! Return the thermodynamic pressure (Pa).
+    /*!
+     *  Since the mass density, temperature, and mass fractions are stored,
+     *  this method uses these values to implement the
+     *  mechanical equation of state \f$ P(T, \rho, Y_1, \dots, Y_K) \f$.
+     *
+     * \f[
+     *    P = \frac{RT}{v-b_{mix}} - \frac{\left(\alpha a\right)_{mix}}{v^2 + 2b_{mix}v - b_{mix}^2 }
+     * \f]
+     *
+     *  where:
+     *
+     * \f[
+     *    \alpha = \left[ 1 + \kappa \left(1-T_r^{0.5}\right)\right]^2
+     * \f]
+     *
+     *  and
+     *
+     * \f[
+     *    \kappa = \left(0.37464 + 1.54226\omega - 0.26992\omega^2\right)                              for omega <= 0.491

```

Comment:
The math formatting on this line and the next one don't quite turn out. You might need to use the `\text{}` macro on the `for omega` bit, and you'll probably want to see if the equations can be aligned with some LaTeX magic.

Reply:
Corrected.